reflex/app.py

```diff
@@ -329,6 +330,12 @@ def add_page(
         # Wrap the component in a fragment.
         component = Fragment.create(component)
 
+        # If component is not a Component object, try to unwrap it into a Fragment
+        if not isinstance(component, Component):
+                with suppress(TypeError):
+                component = Fragment.create(*[i for i in component])
```

Comment:
```suggestion
                component = Fragment.create(*component)
```

no need for the extra list.

Reply:
Ok